src/PowerShellEditorServices.Hosting/Commands/StartEditorServicesCommand.cs

```diff
@@ -298,14 +298,17 @@ private void StartLogging()
 
         private string GetLogDirPath()
         {
-            if (!string.IsNullOrEmpty(LogPath))
+            string logDir = !string.IsNullOrEmpty(LogPath)
+                ? Path.GetDirectoryName(LogPath)
+                : Path.GetDirectoryName(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location));
+
+            // Ensure logDir exists
+            if (!Directory.Exists(logDir))
```

Comment:
No need for this test; [Directory.CreateDirectory() is a no-op if it already exists](https://docs.microsoft.com/en-us/dotnet/api/system.io.directory.createdirectory?view=netframework-4.8)